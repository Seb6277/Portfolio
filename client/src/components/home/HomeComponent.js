import React from 'react'

class HomeComponent extends React.Component{
  render() {
    return(
      <div className="content col-md-11">
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
            labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
            aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum
            dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui
            officia deserunt mollit anim id est laborum.
          </p>
          <img className="home_image" src="https://picsum.photos/300/300" alt="TODO"/>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
            labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
            aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum
            dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui
            officia deserunt mollit anim id est laborum.
          </p>
      </div>
    )
  }
}

export default HomeComponent