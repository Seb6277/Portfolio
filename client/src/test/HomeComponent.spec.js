import React from 'react'
import renderer from 'react-test-renderer'
import {shallow} from 'enzyme'

import HomeComponent from '../components/home/HomeComponent'

describe('<HomeComponent/>', () => {
  it('should render without crash', () => {
    const wrapper = shallow(<HomeComponent/>)
    expect(wrapper)
  });

  it('should match snapshot', () => {
    const component = renderer.create(<HomeComponent/>)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
    tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  });
})