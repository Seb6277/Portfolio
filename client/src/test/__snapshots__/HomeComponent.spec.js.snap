// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeComponent/> should match snapshot 1`] = `
<div
  className="content col-md-10 offset-md-1"
>
  <h1>
    Bienvenue
  </h1>
  <p>
    Bonjour est bienvenue sur mon portfolio ! 
    <br />
    Ici vous trouverez une sélection de mes projets récent ainsi qu'une prise de contactpar formulaire et mon CV.
  </p>
  <img
    alt="TODO"
    className="home_image"
    src="profil.jpg"
  />
  <p>
    A l'aise avec ES6, ES7 et les technologies autour de Javascript (ECMAScript) je touche parfois aussi au développement PHP avec Symfony.
  </p>
  <p>
    Certains projet ici seront des expérimentations, d'autre de vrais site... Vous pouvez toujours suivre mon Github qui dispose de la grande majorité des projets en dépôt publique.
  </p>
  <p>
    Bonne visite !
  </p>
</div>
`;

exports[`<HomeComponent/> should match snapshot 2`] = `
<div
  className="content col-md-10 offset-md-1"
>
  <h1>
    Bienvenue
  </h1>
  <p>
    Bonjour est bienvenue sur mon portfolio ! 
    <br />
    Ici vous trouverez une sélection de mes projets récent ainsi qu'une prise de contactpar formulaire et mon CV.
  </p>
  <img
    alt="TODO"
    className="home_image"
    src="profil.jpg"
  />
  <p>
    A l'aise avec ES6, ES7 et les technologies autour de Javascript (ECMAScript) je touche parfois aussi au développement PHP avec Symfony.
  </p>
  <p>
    Certains projet ici seront des expérimentations, d'autre de vrais site... Vous pouvez toujours suivre mon Github qui dispose de la grande majorité des projets en dépôt publique.
  </p>
  <p>
    Bonne visite !
  </p>
</div>
`;
